docker/resource_docker_container_test.go

```diff
@@ -1100,6 +1100,33 @@ func TestAccDockerContainer_rm(t *testing.T) {
 	})
 }
 
+func TestAccDockerContainer_readonly(t *testing.T) {
+	var c types.ContainerJSON
+
+	testCheck := func(*terraform.State) error {
+		if !c.HostConfig.ReadonlyRootfs {
+			return fmt.Errorf("Container isn't readonly")
+		}
+
+		return nil
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDockerContainerReadOnlyConfig,
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
add `testAccContainerRunning("docker_container.foo", &c),` like in the other tests and it passes ;-) if not  `c`  will be `nil @dubo-dubon-duponey

Reply:
🤦‍♂️
Hiding in shame -  I'll blame... a lapse in judgment :p

Reply:
No worries :) I am also in this club... sometimes hehe